hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/group/GroupService.java

```diff
@@ -3,16 +3,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
+import org.springframework.stereotype.Component;
 import pl.allegro.tech.hermes.api.Group;
 import pl.allegro.tech.hermes.api.helpers.Patch;
+import pl.allegro.tech.hermes.common.query.Query;
 import pl.allegro.tech.hermes.domain.group.GroupNotExistsException;
 import pl.allegro.tech.hermes.domain.group.GroupRepository;
 import pl.allegro.tech.hermes.infrastructure.MalformedDataException;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
-@Service
+@Component
```

Comment:
Yep.. to match all other Services..

Maybe all should be change do @Service.. but they should use the same annotation.